lib/KettleUtils.js

```diff
@@ -116,13 +116,22 @@ kettle.syncFilePromise = function (fileName, message) {
     return togo;
 };
 
+/** @param fileNames {Array of String} An array of filenames to be tested
+ * @return {String} The first member of the array which exists, or `null` if none do
+ */
+kettle.firstExistingFile = function (fileNames) {
```

Comment:
I believe that `kettle.firstExistingFile()` will find directories as well as files (`fs.existsSync(fileName)` will return true for directories). We perhaps should also test the file type (I think `fs.statSync(path)` would be the function to use).